AppCenterAnalytics/AppCenterAnalyticsTests/MSAnalyticsTransmissionTargetTests.m

```diff
@@ -487,4 +510,191 @@ - (void)testEventPropertiesCascading {
   XCTAssertEqual(eventLog.properties[@"g"], @"7777");
 }
 
+- (void)testOverridingCommonSchemaProperties {
```

Comment:
I don't particularly like this style of writing tests.  I would like for this to be split up into more discrete test cases.